portal/src/routes/(authenticated)/projects/[filter=projectSelector]/[id]/+page.server.ts

```diff
@@ -161,10 +165,12 @@ export const actions: Actions = {
     if (
       !form.valid ||
       !form.data.operation ||
-      (!form.data.projects?.length && form.data.projectId === null) ||
-      orgId !== form.data.orgId
-    )
+      (!form.data.projects?.length && form.data.projectId === null)
+    ) {
       return fail(400, { form, ok: false });
+    }
+    // if user modified hidden values
+    if (orgId !== form.data.orgId) return fail(403, { form, ok: false });
```

Comment:
Here we should just use the `orgId` variable and leave `orgId` out of the paging schema.